scripts/update.py

```diff
@@ -215,17 +223,20 @@ def format_df_for_chirp(df: pd.DataFrame) -> pd.DataFrame:
     Returns
     -------
     pd.DataFrame
-        The FM and NBFM repeaters, formatted for CHIRP.
+        The FM repeaters, formatted for CHIRP.
     """
 
-    # Remove footnotes from the Mode column
-    df = df.assign(Mode=df["Mode"].str.replace(r" \[.+\]", "", regex=True))
+    print("Generating generic CHIRP CSV file.")
+
+    total_repeaters = len(df.index)
```

Comment:
```suggestion
    total_repeaters = len(df)
```

Just a tad cleaner, with the same result.

Reply:
Will do